docs/extensions/tutorials/_posts/1970-01-05-Installing3rdPartyPackages.md

```diff
@@ -0,0 +1,323 @@
+
+Doing this will automatically add `react-native-swiper` as a dependency in our `app/package.json` file:
+
+```json{5-7}
+#file: app/package.json
+{
+  "name": "{{ site.example.devName }}.not-swiper",
+  "version": "0.0.1",
+  "description": "Extension that uses react-native-swiper.",
+  "dependencies": {
+    "react-native-swiper": "^1.5.4"
+  }
+}
+```
+
+The other, more preferred way of doing this is to simply add that dependency we see in the `app/package.json` file, which results with `react-native-swiper` being installed when the app is bundled. The reason this is the preferred way of doing it is because the `node_modules` folder generated with our NPM installation is ignored since we utilize the `$ npm pack` command.
+
+### Using the Package
+
+Our extension now has full access to everything `react-native-swiper` has to offer and we can use it just like we would in a normal React Native app, so let's make use of it's simplest example to show how it works. We'll have to edit our `app/screens/SwiperScreen.js` file to use the Swiper by importing the Swiper component, changing the `render();` method to use the Swiper component and making some changes to the `styles` constant so our Swiper component can function properly.
+
+```javascript{12,17-27,38-62}
+#file app/screens/SwiperScreen.js
+import React, {
+  Component
+} from 'react';
+
+import {
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+
+import Swiper from 'react-native-swiper';
+
+export default class SwiperScreen extends Component {
+  render() {
+    return (
+      <Swiper style={styles.wrapper} showsButtons={true}>
+        <View style={styles.slide1}>
+          <Text style={styles.text}>Hello Swiper</Text>
+        </View>
+        <View style={styles.slide2}>
+          <Text style={styles.text}>Beautiful</Text>
+        </View>
+        <View style={styles.slide3}>
+          <Text style={styles.text}>And simple</Text>
+        </View>
+      </Swiper>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  text: {
+    color: '#FFFFFF',
+    fontSize: 30,
+    fontWeight: 'bold',
+  },
+  wrapper: {
+  },
+  slide1: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#9DD6EB',
+  },
+  slide2: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#97CAE5',
+  },
+  slide3: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#92BBD9',
+  }
+});
+```
+
+And there we go, we implemented `react-native-swiper` into our extension. Now we need to push our extension to the Shoutem server and install it onto an app so we can actually test it out. Continuing where we stopped in our terminal:
+
+```ShellSession
+$ cd ..
```

Comment:
You don't need to change to root directory. You can push it from wherever inside of the extension.